week1/project/hackyourweather/.gitignore

```diff
@@ -0,0 +1,22 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+/.pnp
+.pnp.js
+
+# testing
+/coverage
+
+
```

Comment:
I'm not sure why you removed it in the first place, but please add ` /build` back into this file. That folder not meant to be committed to the project.

Reply:
hello samuei .. this was long time ago when i started react module  i send pull req to this repo instead of hackyourhomework repo .. all fixed now